lib/net.js

```diff
@@ -1270,6 +1270,12 @@ function emitListeningNT(self) {
 }
 
 
+function throwIfInvalidPort(port) {
+  if (typeof port !== 'undefined' && !isLegalPort(port))
+    throw new RangeError(`"port" option should be >= 0 and < 65536: ${port}`);
```

Comment:
I would actually recommend refactoring this just a bit and moving it into `internal/net` as a utility method. There are a couple of other places where we check the port and throw if it's not valid that can be updated to use the utility.

```js
function assertPort(port) {
  if (!isLegalPort(port)) 
    throw new RangeError('"port" argument must be >= 0 and < 65536');
}
```
Adding the the `: #{port}` to the end just seems unnecessary to me.